.husky/pre-commit

```diff
@@ -1,4 +1,2 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
-
-yarn build:serverless
```

Comment:
This should be restored because unlike a normal repository with normal source code, this source code must also include the compiled files for the GitHub Action to be useful/up-to-date with the source code. 

This enforces that the compiled code is synced to the latest source code.

Reply:
fixed in latest commits